recipes/vulkan-utility-libraries/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires(f"vulkan-headers/{self.version}", transitive_headers=True)
+
+    def validate(self):
+        check_min_cppstd(self, 17)
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.17.2 <4]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_TESTS"] = False
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def _patch_sources(self):
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
+                        "set(CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE.md", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        copy(self, "Apache-2.0.txt", os.path.join(self.source_folder, "LICENSES"), os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        fix_apple_shared_install_name(self)
+
+    @property
+    def _exported_defines(self):
+        # https://github.com/KhronosGroup/Vulkan-Utility-Libraries/blob/vulkan-sdk-1.3.268.0/src/CMakeLists.txt#L42-L56
+        defines = ["VK_ENABLE_BETA_EXTENSIONS"]
+        if self.settings.os == "Windows":
+            defines += [
+                "NOMINMAX",
+                "WIN32_LEAN_AND_MEAN",
+                "VK_USE_PLATFORM_WIN32_KHR",
+            ]
+        elif self.settings.os == "Android":
+            defines.append("VK_USE_PLATFORM_ANDROID_KHR")
+        elif is_apple_os(self):
+            defines.append("VK_USE_PLATFORM_METAL_EXT")
+            if self.settings.os == "iOS":
+                defines.append("VK_USE_PLATFORM_IOS_MVK")
+            else:
+                defines.append("VK_USE_PLATFORM_MACOS_MVK")
+        return defines
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "VulkanUtilityLibraries")
+
+        # Vulkan::UtilityHeaders
+        self.cpp_info.components["UtilityHeaders"].set_property("cmake_target_name", "Vulkan::UtilityHeaders")
+        self.cpp_info.components["UtilityHeaders"].requires = ["vulkan-headers::vulkanheaders"]
+        self.cpp_info.components["UtilityHeaders"].libdirs = []
+        self.cpp_info.components["UtilityHeaders"].bindirs = []
+
+        # Vulkan::LayerSettings
+        self.cpp_info.components["LayerSettings"].set_property("cmake_target_name", "Vulkan::LayerSettings")
+        self.cpp_info.components["LayerSettings"].requires = ["vulkan-headers::vulkanheaders", "UtilityHeaders"]
```

Comment:
```suggestion
        self.cpp_info.components["LayerSettings"].requires = ["vulkan-headers::vulkanheaders"]
```

Only CompilerConfiguration is required here